TestsUtils/CompileAndExtractTests.lean

```diff
@@ -156,10 +156,21 @@ def typecheckConst (pStore : PureStore) (name : Name) : Except String Unit :=
   | .ok u => .ok u
   | .error err => throw $ toString err
 
-def extractPositiveTypecheckTests (stt : CompileState) : TestSeq :=
-  stt.pStore.consts.foldl (init := .done) fun tSeq const =>
-    tSeq ++ withExceptOk s!"{const.name} ({const.ctorName}) typechecks"
-      (typecheckConst stt.pStore const.name) fun _ => .done
+inductive FoundConstFailure (constName : String) : Prop
+
+instance : Testable (FoundConstFailure constName) :=
+  .isFailure $ .some s!"Could not find constant {constName}"
+
+def extractPositiveTypecheckTests (consts : Option (List Name) := none) (stt : CompileState) : TestSeq :=
```

Comment:
While this is a practical for debugging and iterating locally, I don't think it's a good change for the final code that gets merged. I think that running this test for a particular constant is something that should be adjusted locally and temporarily